src/framework/audio/internal/platform/lin/alsaaudiodriver.h

```diff
@@ -30,11 +30,11 @@
 #include "audiodeviceslistener.h"
 
 namespace muse::audio {
-class LinuxAudioDriver : public IAudioDriver, public async::Asyncable
+class AlsaAudioDriver : public IAudioDriver, public async::Asyncable
```

Comment:
You forgot to rename MUSE_AUDIO_LINUXAUDIODRIVER_H :) But you can also replace it with #pragma once